### DISCUSSION TITLE
Advice for getting the telemetry I want to my radio

### DISCUSSION THREAD

**Discussion by dcan999** [ExpressLRS/ExpressLRS#2729] [Q&A]
Hi Guys. I posted about my project in INAV but it looks like what I want is too complicated (getting telemetry INTO the FC and onto the OSD) so I guess my next approach is trying to get it onto my radio instead, if possible. 

My issue is I want to use ELRS as the control protocol, but the ECU (turbine) only outputs JR/Futaba/HoTT/FRSky protocols. I really want mainly the running state, fuel consumption, RPM and EGT, none of which ELRS (Crossfire) supports as telemetry sensors anyways, so I guess my question is, is there a way to run ELRS protocol for control, but also get telemetry that uses a different protocol to the same radio? I think it would be possible using a second Rx and a PC running Mavlink or something, but I'd really like it right to the radio so I can have audio callouts as well. 

Thanks

**Comment by secretman28** [comment]
sensor need to ba added to the crsf serial to be able to make this, i really want this too i switch to erls from the start, convert all my plane heli and drone i wish to get telemetry for my turbine jet too , im even thinking to get a frsky x20 to replace my tx16 that i love run elrs in external but run my jet in frsky to get full telemetry, fuel capacity egt etc... 

you can use both if you have a edgetx/opentx transmitter use internal for frsky just for telemetry, and use elrs for control but only 1 telemetry can be use .

i try a long time a go to make thing work but not enough people was using it , now elrs have a better on going status for plane part will see  if  they can or not , even made a full blown 16 servo extender baro gyro dual redundant batt 4temps sensor/rpm  just to run elrs in my jet 
was in a bad spot but now doing better project continue will see where time will get me :P
